app/test/ica_test.go

```diff
@@ -0,0 +1,116 @@
+package app_test
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/app/encoding"
+	v1 "github.com/celestiaorg/celestia-app/pkg/appconsts/v1"
+	v2 "github.com/celestiaorg/celestia-app/pkg/appconsts/v2"
+	"github.com/celestiaorg/celestia-app/test/util"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/x/params/types/proposal"
+	icahosttypes "github.com/cosmos/ibc-go/v6/modules/apps/27-interchain-accounts/host/types"
+	"github.com/stretchr/testify/require"
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/libs/log"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	version "github.com/tendermint/tendermint/proto/tendermint/version"
+	dbm "github.com/tendermint/tm-db"
+)
+
+// TestICA verifies that the ICA module's params are overridden during an
+// upgrade from v1 -> v2.
+func TestICA(t *testing.T) {
+	testApp, _ := setupTestApp(t, 3)
+	supportedVersions := []uint64{v1.Version, v2.Version}
+	require.Equal(t, supportedVersions, testApp.SupportedVersions())
+
+	ctx := testApp.NewContext(true, tmproto.Header{
+		Version: version.Consensus{
+			App: 1,
+		},
+	})
+	testApp.BeginBlock(abci.RequestBeginBlock{Header: tmproto.Header{
+		Height:  2,
+		Version: version.Consensus{App: 1},
+	}})
+
+	// app version should not have changed yet
+	require.EqualValues(t, 1, testApp.AppVersion())
+
+	// global min gas price should not have been set yet
```

Comment:
I think this comment needs to be updated

Reply:
6c016b4a6cc730d29e8834e0e293d54f75bc3d8b